intel_extension_for_transformers/llm/runtime/graph/core/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ find_package(Threads REQUIRED)
 file(GLOB layers_srcs "layers/*.cpp")
 set(sources ne_layers.c ${layers_srcs})
 
-add_library_w_warning(ne_layers "${sources}")
+add_library_w_warning(ne_layers SHARED "${sources}")
```

Comment:
hard-code change will not generate any exported symbols on Windows. It's not related to BUILD_SHARED_LIBS

Reply:
Thanks! 3918179 should work now.